plugins/modules/pnp_intent.py

```diff
@@ -728,13 +753,16 @@ class instance for further use.
                     function='get_device_list',
                     params={"serial_number": device["deviceInfo"]["serialNumber"]}
                 )
-
+                self.log("Device details for the serial number {0}\
+                    obtained from the API 'get device list'\
+                        are {1}".format(device["deviceInfo"]["serialNumber"], str(multi_device_response)), "DEBUG")
                 if (multi_device_response and (len(multi_device_response) == 1)):
                     devices_added.append(device)
-
+                    self.log("Details of the added device is {0}".format(str(device)), "INFO")
```

Comment:
self.log("Details of the added device: {0}".format(str(device)), "INFO")

Reply:
Resolved